index.js

```diff
@@ -326,11 +326,12 @@ class EmeEncryptionSchemePolyfillMediaKeySystemAccess {
  */
 EmeEncryptionSchemePolyfill.originalRMKSA_;
 
-// Support for CommonJS module format.
+// Support for CommonJS and AMD module formats.
 /** @suppress {undefinedVars} */
 (() => {
-  if (typeof exports !== 'undefined') {
+  // eslint-disable-next-line no-undef
```

Comment:
Alternatively you could declare that `module` is a global variable:
https://eslint.org/docs/user-guide/configuring#specifying-globals

Reply:
Nice!  Thanks for the tip.  Done.